packages/icon/package.py

```diff
@@ -102,6 +105,9 @@ class Icon(Package):
     variant('silent-rules',
             default=True,
             description='Build with Make silent rules ON')
+    variant('use-standard-account',
```

Comment:
I think this should not be a variant, because it's not a variation of the package. The program is not affected by it.
It's a flag that tells CSCS who to bill. Or am I mistaken?

Reply:
It changes the account that is put into the runscript, see https://github.com/C2SM/icon/blob/master/run/create_target_header#L921. I agree that this is not a variant by definition. We need to find a clever way to pass a user-defined account name. But this can be content of another PR.

Reply:
OK then I remove this variant again and run the tests again.